website/versioned_docs/version-0.55/animated.md

```diff
@@ -195,14 +195,16 @@ Config is an object that may have the following options.
 
 Note that you can only define one of bounciness/speed, tension/friction, or stiffness/damping/mass, but not more than one:
 
+<!-- alex ignore rule -->
```

Comment:
Instead of ignoring the rule can you change the below to \[\`Facebook Pop\`\]?

Reply:
Thanks, I'll do that!